### ISSUE TITLE
How can I get the sort name

### ISSUE THREAD

**Issue by ChengYaoLiang** [rstaib/jquery-bootgrid#111]
The post body always like this:
current=1&rowCount=10&sort[sender]=asc&searchPhrase=
I think I can get the sort parameter by for each the request parameters ...But it looks silly~~
I use Java server platform,I have no idea how to direct access to a dynamic parameter.
How could I get the sort name(sender) on the web side in a simple way ?
I see no one ask for this~~~am I so stupid 〒_〒

**Comment by nicholascowan** [comment]
Create a requestHandler which maps it to whatever you want.

```javascript
requestHandler: function (request) {
  if (request.sort) {
    request.sortBy = Object.keys(request.sort)[0]; //this only gets first sort param
    request.sortDir = request.sort[request.sortBy];
    delete request.sort
  }
  return request;
}
```

**Comment by ChengYaoLiang** [comment]
This looks really great.
I do appreciate your timely help.

**Comment by HarveyEV** [comment]
Yeah, I had the same question. Thanks for the handler. Wondering why it doesn't look like a simple nv pair:

```
current=1&rowCount=10&sortby=sender&dir=asc&searchPhrase=
```

That would be easier to handle on the server side and not force us to write a custom handler

**Comment by lihua123569** [comment]
requestHandler: function (request) {
  if (request.sort) {
    request.sortBy = Object.keys(request.sort)[0]; //this only gets first sort param
    request.sortDir = request.sort[request.sortBy];
    delete request.sort
  }
  return request;
}

how to get more sort params????

**Comment by legendjslc** [comment]
How do I write it in python?
